src/angular-zendesk-widget/zendeskWidget.service.js

```diff
@@ -42,7 +42,9 @@
 
           angular.forEach(apiMethods, function(method) {
             ZendeskWidgetApi.prototype[method] = function() {
-              $window.zE[method].apply($window.zE, arguments);
+              $window.zE(function() {
+                $window.zE[method].apply($window.zE, arguments);
```

Comment:
This is a bug: `arguments` now blanks out due to the closure. Think you need to do something like this:

```js
            ZendeskWidgetApi.prototype[method] = function() {
            var closureArgs = arguments;
              $window.zE(function() {
                $window.zE[method].apply($window.zE, closureArgs);
              })
            };
```